README.md

```diff
@@ -1,6 +1,20 @@
 [![ROS Industrial
 CI](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/actions/workflows/industrial_ci_action.yml/badge.svg?branch=master&event=push)](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/actions)
 
+---
+##### ++ Deprecation Notice ++
```

Comment:
Why so small
```suggestion
## ++ Deprecation Notice ++
```

Reply:
Alright. Less likely to oversee that way.